db/migration/V022__insert_sample_assessment_data.sql

```diff
@@ -0,0 +1,163 @@
+INSERT INTO `activity_types`
+  (`title`)
+VALUES
+  ("quiz"),
+  ("test");
+
+INSERT INTO `assessment_submission_states`
+  (`title`)
+VALUES
+  ("Inactive"),
+  ("Active"),
+  ("Opened"),
+  ("In Progress"),
+  ("Expired"),
+  ("Submitted"),
+  ("Graded"),
+  ("Hidden");
+
+INSERT INTO `program_participant_roles`
+  (`title`)
+VALUES
+  ("Facilitator"),
+  ("Participant");
+
+INSERT INTO `curriculums`
+  (`title`, `principal_id`)
+VALUES
+  ("Beginner Web Development", 2);
+
+INSERT INTO `programs`
+  (`title`, `start_date`, `end_date`, `time_zone`, `principal_id`, `curriculum_id`)
+VALUES
+  ("Spring 2023", "2023-02-06", "2023-03-31", "America/Vancouver", 2, 3);
```

Comment:
Hello! I just wanted to check if you wanted to maybe write the dates as MM/DD/YYYY or DD/MM/YYYY as is the convention for US / Canada?

Reply:
Thanks for reviewing, I feel that this format was chosen because of SQL formatting rule. It should be like TIMESTAMP with the format  YYYY-MM-DD HH:MM:SS.